docs/about/concepts/audio/quality-metrics.md

```diff
@@ -0,0 +1,297 @@
+personas: ["data-scientist-focused", "mle-focused"]
+difficulty: "beginner"
+content_type: "concept"
+modality: "audio-only"
+---
+
+(about-concepts-audio-quality-metrics)=
+# Audio Quality Metrics
+
+Understanding the quality metrics used in NeMo Curator for evaluating speech transcription accuracy, audio characteristics, and overall dataset quality.
+
+## Transcription Accuracy Metrics
+
+### Word Error Rate (WER)
+
+The primary metric for measuring ASR transcription quality:
+
+**Definition**: Percentage of words that differ between ground truth and predicted transcriptions.
+
+**Calculation**: 
+```
+WER = (Substitutions + Deletions + Insertions) / Total_Words × 100
+```
+
+**Interpretation**:
+- **WER = 0%**: Perfect transcription match
+- **WER ≤ 10%**: Excellent quality (production-ready)
+- **WER ≤ 25%**: Good quality (suitable for most training)
+- **WER ≤ 50%**: Moderate quality (may need review)
+- **WER > 75%**: Poor quality (consider filtering)
+
+**Example**:
+```python
+# Ground truth: "hello world example"
+# Prediction:   "hello word example"
+# WER = 1/3 × 100 = 33.33% (1 substitution out of 3 words)
+```
+
+Implementation reference: `nemo_curator/stages/audio/metrics/get_wer.py:23-29` (utility) and `nemo_curator/stages/audio/metrics/get_wer.py:48-72` (stage)
+
+::::{note}
+WER and CER utilities depend on the `editdistance` package.
+::::
+
+### Character Error Rate (CER)
+
+More granular accuracy measurement at the character level:
+
+**Definition**: Percentage of characters that differ between ground truth and predicted transcriptions.
+
+**Calculation**:
+```
+CER = (Character_Substitutions + Character_Deletions + Character_Insertions) / Total_Characters × 100
+```
+
+**Use Cases**:
+- Languages with complex morphology
+- Detailed accuracy analysis
+- Character-level model evaluation
+
+**Example**:
+```python
+# Ground truth: "hello"
+# Prediction:   "helo" 
+# CER = 1/5 × 100 = 20% (1 deletion out of 5 characters)
+```
+
+Implementation reference: `nemo_curator/stages/audio/metrics/get_wer.py:32-35`
+
+## Audio Characteristic Metrics
+
+### Duration Analysis
+
+**Audio Duration**: Precise measurement of audio file length in seconds.
+
+Implementation reference: `nemo_curator/stages/audio/common.py:43-68` (`GetAudioDurationStage` reads audio using `soundfile` and sets seconds as frames ÷ sample rate; sets -1.0 on error)
+
+**Speech Rate Metrics**:
+- **Words per Second**: `word_count / duration`
+- **Characters per Second**: `character_count / duration`
+
+Implementation reference: `nemo_curator/stages/audio/metrics/get_wer.py:38-45` (`get_wordrate`, `get_charrate`)
+
+**Quality Indicators**:
+- **Optimal Speech Rate**: 2-4 words per second for most languages
+- **Duration Range**: 1-30 seconds optimal for ASR training
+- **Very Short Audio** (< 0.5s): Often incomplete utterances
+- **Very Long Audio** (> 60s): May contain multiple speakers or topics
```

Comment:
I am curious where these numbers come from?

Reply:
They were suggested during doc gen. If we'd like to suggest our own I can swap them out but for now i've removed them all from the audio docs